explorer/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```

Comment:
I know that this is just the output of `create-react-app`, but I don't think that `.DS_Store` should be here. Instead, it should be in a global gitignore:
```sh
$ echo '.DS_Store' >> ~/.gitignore_global
$ git config --global core.excludesFile ~.gitignore_global
```
There are dual advantages:
  * your global gitignore, with your platform/editor/IDE-specific bindings, persists across all projects;
  * when ten different developers on ten different platforms/editor/IDEs start contributing to the project, the project's `.gitignore` doesn't get polluted.

For instance, here's mine: https://gist.github.com/wchargin/868e0147911c0486a751ad99a71133fb